### PR TITLE
add vgg16_aws_dist

### PR DESCRIPTION
**Pull request by putcn**
add vgg16_aws_dist.
this test will be performed in aws utilizing [aws_runner](https://github.com/putcn/aws_runner/) to dynamically allocating instances, network resources, collect metrics data and save results to CE for further analytics. 
Testing models are using [benchmar/fluid](https://github.com/PaddlePaddle/Paddle/tree/develop/benchmark/fluid). this case is running vgg16 dist train test, but it's very easy to add more round of tests with other models and cluster configs. these configs can be updated in run.xsh and continuous_evaluation.py

Please note: nccl2 is not yet supported, will add it in next PR

### CODE REVIEW

**Comment by Superjomn** [comment]
看了代码格式，貌似没有跑 `pre-commit` ?

**Comment by putcn** [comment]
跑了啊, 

```
paddle-ce-latest-kpis git:(vgg16_aws_dist) ✗ pre-commit run -a
yapf.....................................................................Passed
Check for added large files..............................................Passed
Check for merge conflicts................................................Passed
Check for broken symlinks................................................Passed
Fix End of Files.........................................................Passed
```

File: vgg16_aws_dist/ce_runner.py [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+                }
+                for metric_name, _ in collector.store.iteritems():
+                    if metric_name not in fieldnames:
+                        fieldnames.append(metric_name)
+                    row[metric_name] = collector.avg(metric_name)
+                    rows.append(row)
+            writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+            writer.writeheader()
+            for row in rows:
+                writer.writerow(row)
```
Comment (Superjomn) [inline vgg16_aws_dist/ce_runner.py]: add an empty line here.

**Comment by putcn** [comment]
@Yancey1989 currently we can't, because the metric retrieve part is different, we have to output specific format of metrics so that aws_runner can identify and catch these values.

File: vgg16_aws_dist/run.xsh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+echo "going to start testing"
+# start aws testingr
+python ce_runner.py \
+    --key_name aws_benchmark_us_east \
+    --security_group_id sg-95539dff \
+    --online_mode yes \
+    --pserver_command $training_command \
+    --trainer_command $training_command \
+    --docker_image $fluid_benchmark_dockerhub_tag
```
Comment (Yancey1989) [inline vgg16_aws_dist/run.xsh]: Seems need a blank line at the end of the file.

File: vgg16_aws_dist/speedup_vgg_16_1_1_0_factor.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+[1.0]
```
Comment (Yancey1989) [inline vgg16_aws_dist/speedup_vgg_16_1_1_0_factor.txt]: Can we remove these KPI result files?

Comment (putcn) [reply vgg16_aws_dist/speedup_vgg_16_1_1_0_factor.txt]: no, CE have to have initial KPI data

File: vgg16_aws_dist/run.xsh (hunk 2 above)
Comment (putcn) [reply vgg16_aws_dist/run.xsh]: sure, will update

**Pull request merged**